eval/nki_docs_parsing_scripts/get_op_list.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+def copy_alternate_lines(input_file, output_file):
+    """
+    Copies every other line from input_file to output_file.
```

Comment:
Why the every other line logic? I'd understand just wanting to drop extraneous content, but hinging the logic around every other line seems like you might miss a lot.